static/directives/repo-view/repo-panel-mirroring.html

```diff
@@ -361,124 +361,125 @@ <h4>Status</h4>
+      </form>
+    </div>
 
-  <!-- Modal: Change mirroring sync interval -->
-  <div class="cor-confirm-dialog"
-       dialog-context="vm.syncIntervalChanges"
-       dialog-action="vm.changeConfig(info, callback)"
-       dialog-title="Update next synchronization date"
-       dialog-action-title="Update"
-       dialog-form="context.mirroringform">
-    <form name="context.mirroringform" class="co-single-field-dialog">
-      <label>Synchronization Interval</label>
-      <div>
-        <span class="interval-input" seconds="vm.syncIntervalChanges.values.sync_interval"></span>
-      </div>
-    </form>
-  </div>
+    <!-- Modal: Change mirroring sync interval -->
+    <div class="cor-confirm-dialog"
+        dialog-context="vm.syncIntervalChanges"
+        dialog-action="vm.changeConfig(info, callback)"
+        dialog-title="Update next synchronization date"
+        dialog-action-title="Update"
+        dialog-form="context.mirroringform">
+      <form name="context.mirroringform" class="co-single-field-dialog">
+        <label>Synchronization Interval</label>
+        <div>
+          <span class="interval-input" seconds="vm.syncIntervalChanges.values.sync_interval"></span>
+        </div>
+      </form>
+    </div>
 
-  <!-- Modal: Change external repository -->
-  <div class="cor-confirm-dialog"
-       dialog-context="vm.externalRepositoryChanges"
-       dialog-action="vm.changeConfig(info, callback)"
-       dialog-title="Update external repository"
-       dialog-action-title="Update"
-       dialog-form="context.mirroringform">
-    <form name="context.mirroringform" class="co-single-field-dialog">
-      External Repository
-      <input type="text"
-             class="form-control"
-             ng-model="vm.externalRepositoryChanges.values.external_reference">
-    </form>
-  </div>
+    <!-- Modal: Change external repository -->
+    <div class="cor-confirm-dialog"
+        dialog-context="vm.externalRepositoryChanges"
+        dialog-action="vm.changeConfig(info, callback)"
+        dialog-title="Update external repository"
+        dialog-action-title="Update"
+        dialog-form="context.mirroringform">
+      <form name="context.mirroringform" class="co-single-field-dialog">
+        External Repository
+        <input type="text"
+              class="form-control"
+              ng-model="vm.externalRepositoryChanges.values.external_reference">
+      </form>
+    </div>
 
-  <!-- Modal: Change Credentials -->
-  <div class="cor-confirm-dialog"
-       dialog-context="vm.credentialsChanges"
-       dialog-action="vm.changeConfig(info, callback)"
-       dialog-title="Update Credentials"
-       dialog-action-title="Update"
-       dialog-form="context.mirroringform">
-    <form name="context.mirroringform" class="co-single-field-dialog">
-      <label>Username</label>
-      <input type="text"
-             class="form-control"
-             ng-model="vm.credentialsChanges.values.external_registry_username">
```

Comment:
This is the core fix: using the transcluded scope instead of the outer controller scope.